pkg/volume/iscsi/iscsi_util.go

```diff
@@ -417,10 +420,10 @@ func (util *ISCSIUtil) AttachDisk(b iscsiDiskMounter) (string, error) {
 				devicePath = strings.Join([]string{"/dev/disk/by-path/pci", "*", "ip", tp, "iscsi", b.Iqn, "lun", b.Lun}, "-")
 			}
 
-			if exist := waitForPathToExist(&devicePath, multipathDeviceTimeout, iscsiTransport); !exist {
-				klog.Errorf("Could not attach disk: Timeout after %ds", multipathDeviceTimeout)
+			if exist := waitForPathToExist(&devicePath, multipathSecondPathDiscoveryTimeout, iscsiTransport); !exist {
```

Comment:
`multipathSecondPathDiscoveryTimeout` -> `deviceDiscoveryTimeout`, it is used for all paths, even the first one.

Reply:
@jsafrane updated the patchset with new variable name. Thanks!